src/universal/config/routes.ts

```diff
@@ -31,9 +31,6 @@ export const AppRoutes: Record<string, string> = {
   BUURT: '/buurt',
   BEZWAREN: '/bezwaren',
   'BEZWAREN/DETAIL': '/bezwaren/:uuid',
-  API_LOGIN: '/api/login',
-  API1_LOGIN: '/api1/login',
-  API2_LOGIN: '/api2/login',
```

Comment:
Waarom heb je deze weggehaald?

Reply:
Ik ze niet terug vinden in de `src` folder. Maar zag ze later staan in de ngnix config en heb ze daarom terug gezet.